Docker build error

**Describe the bug**
Docker build results in build error due to syntax error resulting from python:

```terminal
SyntaxError: invalid syntax
  File "/source/src/../tools/tflite2c.py", line 57
    print('  ', end='')
               make[1]: *** [Makefile:19: /source/src/build/lib/libtf/libtf_builtin_models.h] Error 1
```

**To Reproduce**
Steps to reproduce the behavior:
```bash
git clone https://github.com/openmv/openmv.git --depth=50
cd openmv/docker
make TARGET=OPENMV4
```

**Expected behavior**
Builds without error. 

**Screenshots**
![Screenshot from 2022-07-18 13-48-01](https://user-images.githubusercontent.com/100287193/179505040-b9808c2d-a4a0-4211-a6a1-5ad0ceeee89e.png)

**System information**
 - OS: Ubuntu 2022.2
 - Docker Client Version: 20.10.17

**Questions** 
Is there a wrong python version being used by docker?

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -5,7 +5,7 @@ ENV ARCH $DAPPER_HOST_ARCH
 ENV DAPPER_OUTPUT ./docker/build
 ENV DAPPER_DOCKER_SOCKET true
 ENV DAPPER_TARGET dapper
-RUN apt update && apt install -y build-essential wget git python3 python
+RUN apt update && apt install -y build-essential wget git python3 python-is-python3
 ENV GCC_URL="https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-x86_64-linux.tar.bz2" 
 RUN mkdir ./gcc
 RUN wget --no-check-certificate -O - ${GCC_URL} | tar --strip-components=1 -jx -C ./gcc

